modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc
+:_content-type: PROCEDURE
+
+[id="op-using-custom-scc-to-improve-prod-security_{context}"]
+= Using a custom SCC to improve pod security
+
+To improve the security of your builds and runs, you can use custom security context constraints (SCCs).
+
+.Procedure
+
+Create a `pipelines-scc` Security Context Constraints (SCC) and configure its `runAsUser` and `seLinuxContext` fields.
++
+.Example `pipelines-scc` object
+[source,yaml,subs="attributes+"]
+----
+...
+runAsUser:
+  type: MustRunAs
+  uid: 1000
+seLinuxContext:
+  type: MustRunAs
+...
+----
+
+[NOTE]
+====
+The `pipeline` service account allows images to run only as user `1000`, not as root.
+====
+
+You can use the following SCC configuration as reference:
```

Comment:
```suggestion
```
Consider removing this for minimalist content. It's already said in the following example heading, and the user doesn't need it to be repeated, imo.